### DISCUSSION TITLE
Creating a Contract Call + NFT Weighting Strategy

### DISCUSSION THREAD

**Discussion by ruvnet** [snapshot-labs/snapshot#3341] [Q&A] [answered]
Hi Everyone, we're launching a new NFT Gift Card protocol & DAO (CardFi.co) that allows you to associate ERC20 tokens with specific NFTs (erc721) where the current holder of the NFT can then redeem the balance. 

We're also using this as the basis of our DAO membership. We'd like to combine an NFT weighting strategy, with a contract call that checks the balance of the current holder of the DOA membership using our smart contract and gives them a combined vote based on the weighting of the membership NFT card (Gold, Bronze, etc) and the number of CardFi erc20 tokens held in our smart contract. 

I see each of these strategies is possible independently, but nothing has been created that combines both strategies. I'm looking for some guidance. 

@rUv

**Comment by ChaituVR** [comment, marked as answer]
Hi @ruvnet You can add the independent strategies to your space and result will be sum of those strategies, so a new strategy is not required in this case :)
